lib/front-end-builds-notifier.js

```diff
@@ -1,107 +1,104 @@
-var CoreObject = require('core-object');
```

Comment:
👇 this file is important, and has been changed/refactored quite a bit so needs a good 👀

Reply:
Gave this file the most attention 🦅